src/modality-specific-files/near-infrared-spectroscopy.md

```diff
@@ -214,9 +214,9 @@ Note that upper-case is REQUIRED.
 | NIRSCWHBO                   | Continuous wave oxygenated hemoglobin (oxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbO in SNIRF.                                                  |
 | NIRSCWHBR                   | Continuous wave deoxygenated hemoglobin (deoxyhemoglobin) concentration measurements. Equivalent to dataTypeLabel HbR in SNIRF.                                              |
 | NIRSCWMUA                   | Continuous wave optical absorption  measurements. Equivalent to dataTypeLabel mua in SNIRF.                                                                                  |
-| ACCEL                       | Accelerometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z). |
-| GYRO                        | Gyrometer channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z).     |
-| MAGN                        | Magnetomenter channel, one channel for each orientation. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z). |
+| ACCEL                       | Accelerometer channel, one channel for each spatial axis. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z). |
+| GYRO                        | Gyrometer channel, one channel for each spatial axis. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z).     |
+| MAGN                        | Magnetomenter channel, one channel for each spatial axis. An extra column `component` for the axis of the orientation MUST be added to the `*_channels.tsv` file (x, y or z). |
```

Comment:
Look at the failing check: https://github.com/bids-standard/bids-specification/actions/runs/4162663874/jobs/7202110297#step:5:29

It says:

> 217:207-217:208  warning  Misaligned table fence  table-pipe-alignment  remark-lint

which tells you that you need to align the markdown table fences. They should look like this:

```
| a neat table |
| ------------ |
| hi           |
```

instead of

```
| a messy table |
| ----------- |
| hi        |
```

This is most easily done in an editor in which you switch off "line wrapping" (or "word wrapping"), so that each line in the file is shown in an actual line on the display

Reply:
Alright thanks a lot for the pointers. Not entirely sure about how nirs.yaml in "checks" folder is used but it's updated as I think it should be, and the checks are all passing now.